Description: switch test from ava to tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-04-01

--- a/test.js
+++ b/test.js
@@ -1,12 +1,14 @@
-import test from 'ava';
+import {default as test} from 'tape';
 import {pathExists, pathExistsSync} from './index.js';
 
 test('async', async t => {
 	t.true(await pathExists('test.js'));
 	t.false(await pathExists('fail'));
+	t.end();
 });
 
 test('sync', t => {
 	t.true(pathExistsSync('test.js'));
 	t.false(pathExistsSync('fail'));
+	t.end();
 });
